jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonSamProjectWizard.kt

```diff
@@ -51,7 +51,10 @@ abstract class PythonSamProjectTemplate : SamAppTemplateBased() {
         indicator: ProgressIndicator
     ) {
         super.postCreationAction(settings, contentRoot, rootModel, indicator)
-        addSourceRoots(rootModel.project, rootModel, contentRoot)
+        if(settings.packagingType != PackageType.IMAGE) {
```

Comment:
image templates still technically have source code (and therefore source roots). should we read out the docker context and use that instead?

Reply:
See `software.aws.toolkits.jetbrains.services.lambda.sam.SamTemplateUtils#getCodeLocation`

The current method it is using (`software.aws.toolkits.jetbrains.services.lambda.sam.SamCommon.Companion#getCodeUrisFromTemplate`) should be migrated off of and deleted

Reply:
Migrated off getCodeUrisFromTemplate from this function, will add  a separate PR for deleting it completely